code/game/dna/mutations/mutation_powers.dm

```diff
@@ -428,7 +428,27 @@
 				return
 			user.visible_message("<span class='danger'>[user] [pick("chomps","bites")] off [the_item]'s [limb]!</span>")
 			playsound(user.loc, 'sound/items/eatfood.ogg', 50, 0)
-			limb.droplimb(0, DROPLIMB_SHARP)
+			if(istype(limb,/obj/item/organ/external/groin))
+				limb.receive_damage(100, sharp = TRUE)
+				// Groin won't actually drop, but this spills
+				// out the organs that were in it.
+				limb.droplimb(FALSE, DROPLIMB_SHARP)
+
+				var/obj/item/organ/external/left_leg = H.get_organ(BODY_ZONE_L_LEG)
+				if(istype(left_leg))
+					left_leg.droplimb(FALSE, DROPLIMB_SHARP)
+
+				var/obj/item/organ/external/right_leg = H.get_organ(BODY_ZONE_R_LEG)
+				if(istype(right_leg))
+					right_leg.droplimb(FALSE, DROPLIMB_SHARP)
+
+				var/obj/item/organ/external/chest = H.get_organ(BODY_ZONE_CHEST)
+				if(istype(chest))
+					chest.receive_damage(50, sharp = TRUE)
+			else
+				limb.droplimb(FALSE, DROPLIMB_SHARP)
```

Comment:
Could you maybe invert this if-statement?

Reply:
Yup, that worked. Done.